apps/vmq_server/src/vmq_schema.erl

```diff
@@ -532,7 +532,7 @@ extract(Prefix, Suffix, Val, Conf) ->
     NameSubPrefix = lists:flatten([Prefix, ".$name"]),
     [
         begin
-            Addr = parse_addr(StrAddr),
+            {ok, Addr} = parse_addr(StrAddr),
```

Comment:
We need to add `{ok, Addr} = parse_addr(StrAddr)` to the `extract_var` function (added by @mths1), a couple of lines above.
This fixes the schema generation issue.